go.sum

```diff
@@ -1,7 +1,12 @@
+fyne.io/fyne v1.4.3 h1:356CnXCiYrrfaLGsB7qLK3c6ktzyh8WR05v/2RBu51I=
+github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
+github.com/Jacalz/wormhole-gui v1.3.0 h1:rZrYxtNp8LS8XCvlB13oeeJMIGvuhqcsNDqmKDrt6zE=
```

Comment:
Just a few extra `go.sum` entries leaked in :)

Reply:
Good catch!

Reply:
No idea how those got there (can't remember getting wormhole-william), but should be fixed now. Nice catch!